npc/pre-re/merchants/hair_dyer.txt

```diff
@@ -37,6 +37,7 @@ prt_in,243,168,4	script	Jovovich	91,{
 	next;
 	while(1) {
 		switch(select("Dye Hair:Tips and Information:Cancel")) {
+			clear;
```

Comment:
Please use standard format for your next work.
The issue is from
```
			mes "[Hairdresser Jovovich]";
			mes "* Fees and Information *";
			mes "- 1 Dyestuffs item of the color of hair you want.";
			mes "- 1000 zeny fee.";
			break;
```
There should be a "close;" instead of "break"